src/app/payment/paymentMethod/GooglePayPaymentMethod.spec.tsx

```diff
@@ -89,7 +89,35 @@ describe('when using Google Pay payment', () => {
                 [method.id]: {
                     walletButton: 'walletButton',
                     onError: defaultProps.onUnhandledError,
+                    onPaymentSelect: expect.any(Function),
                 },
             }));
     });
+
+    it('reinitializes method once payment option is selected', async () => {
+        const container = mount(<PaymentMethodTest { ...defaultProps } method={ method } />);
+        const component: ReactWrapper<WalletButtonPaymentMethodProps> = container.find(WalletButtonPaymentMethod);
+
+        component.prop('initializePayment')({
+            methodId: method.id,
+            gatewayId: method.gateway,
+        });
+
+        const options: PaymentInitializeOptions = (checkoutService.initializePayment as jest.Mock).mock.calls[0][0];
+
+        // tslint:disable-next-line:no-non-null-assertion
```

Comment:
Could we avoid a hardcoded assertion, `tslint:disable-next-line:no-non-null-assertion`?